### PR TITLE
[Icons]: Update icons from Figma

### PR DESCRIPTION
**Pull request by brave-support-admin**
> @brave/leo@0.0.1 update-icons
> tsx ./src/scripts/update-icons.ts

Ignoring icon rainbow-color.svg because:
- has a raster image element
Ignoring icon bitcoin.com-color.svg because:
- has a raster image element
Ignoring icon daum-color.svg because:
- has a raster image element

### CODE REVIEW

File: icons/toggle-android.svg
Comment (AlanBreck) [inline icons/toggle-android.svg:1]: @fallaciousreasoning, do you know if this is going to cause a problem in `brave-core`?

**Review by AlanBreck — APPROVED** [review]

File: icons/toggle-android.svg
Comment (fallaciousreasoning) [reply icons/toggle-android.svg:1]: Unused in brave-core :relieved:

File: icons/desktop-vpn-error-color.svg
Comment (fallaciousreasoning) [inline icons/desktop-vpn-error-color.svg:1]: @aguscruiz are these color changes intentional? They look a bit less vibrant

Comment (aguscruiz) [reply icons/desktop-vpn-error-color.svg:1]: I don't know why they changed, I haven't changed those in Figma. I've only added a couple icons but nothing on the brands

Comment (aguscruiz) [reply icons/desktop-vpn-error-color.svg:1]: The .diff explorer is crashing my browser lol, I can't even look at the differences

**Review by fallaciousreasoning — APPROVED** [review]

**Pull request merged**